.github/workflows/conformance-clustermesh.yaml

```
@@ -0,0 +1,504 @@
+name: ClusterMesh
```

Let's add the trigger phrase to the test name so it's easier to find from a PR's actions status overview.

```suggestion
name: ClusterMesh (ci-clustermesh)
```

---

We should add some linter for this. I always forget.